charts/ibm-cpd-wkc-prereqs/templates/setkernelparams.yaml

```diff
@@ -0,0 +1,55 @@
+apiVersion: apps/v1
```

Comment:
This is required on IBM Cloud because of the RHEL/CoreOS issue. On non-ROKS clusters we should use the node tuning operator to make these configuration changes

Reply:
We will fix this separately as an issue